doc/camp_tutorial/boot_camp/part_1_code/box_model.F90

```diff
@@ -79,8 +90,11 @@ program box_model
                      camp_state%state_var( idx_O2  )
   end do
 
-  deallocate( camp_core )
```

Comment:
I tried that line in my last commit and is working fine, so It was fixed somewhere in the way. I will push that last commit with the rest of the changes.

I think it was failing because a missing "  call camp_mpi_finalize( )" or something like that